galen-core/src/main/java/com/galenframework/validation/specs/SpecValidationText.java

```diff
@@ -36,6 +37,7 @@ public ValidationResult check(PageValidation pageValidation, String objectName,
         
         Rect area = mainObject.getArea();
         String realText = mainObject.getText();
+        realText = new String(realText.getBytes(Charset.forName("utf-8")));
```

Comment:
Could you please move this line down after null pointer check? Because it doesn't make sense to check for null after the variable was already used.